Yoast/YoastSEO.js
# Disable markers Text competing links assessment

## Summary

This PR can be summarized in the following changelog entry:

* Removes eye-markers from the assessment that checks if the text contains links whose link text contains keyphrase

## Relevant technical choices:

* The eye-markers are disabled because marking hrefs does not work properly. For two ways that would allow to fix this issue using the current html parser please consult #1897 

## Test instructions

This PR can be tested by following these steps:

* Create a text and a keyphrase. Add a link with link text that contains keyphrase.
* The text competing links assessment should return a warning, but no eye marker should be available.
* Remove the keyphrase from the link. The text competing links assessment warning should disappear.
* Check if markers are still present for other assessments.

Fixes #1897
